logstash-core/lib/logstash/environment.rb

```diff
@@ -1,7 +1,39 @@
 # encoding: utf-8
 require "logstash/errors"
+require "logstash/config/cpu_core_strategy"
+require "logstash/settings"
 
 module LogStash
+
+  [
+            Setting::String.new("node.name", Socket.gethostname),
+            Setting::String.new("config.path", nil, false),
```

Comment:
@jordansissel I was just about to comment on this bit :) Lets make this `path.config` to be consistent